azure-iot-hub-devicesdk/samples/send_telemetry_device.py

```diff
@@ -0,0 +1,52 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import os
+from threading import Timer
+from azure.iot.hub.devicesdk.device_client import DeviceClient
+from azure.iot.hub.devicesdk.auth.authentication_provider_factory import from_connection_string
+
+# The connection string for a device should never be stored in code. For the sake of simplicity we're using an environment variable here.
+conn_str = os.getenv("IOTHUB_DEVICE_CONNECTION_STRING")
+# The "Authentication Provider" is the object in charge of creating authentication "tokens" for the device client.
```

Comment:
>"tokens" [](start = 83, length = 8)

putting this in quotes makes it lose credibility.  I would maybe say: "The authentication provider is the object that manages the connection credentials for the device client."  I don't like the word "manages" either, but I'm not sure how else to phrase it.